src/components/common/Navbars/NavBar.js

```diff
@@ -100,20 +102,20 @@ function NavBar(props) {
           )}
           <NavLink to="/login">
             <button
-              className={`navbar__btn navbar__login ${
-                localStorage.getItem('okta-token-storage') ? 'navbar__hide' : ''
-              }`}
+            // className={`navbar__btn navbar__login ${
```

Comment:
Let's just remove all these attributes from the button instead of commenting them out.